vendor/engines/split_accounts/app/models/concerns/account_extension.rb

```diff
@@ -0,0 +1,11 @@
+module Concerns
```

Comment:
I would put this under the SplitAccounts namespace. And the `Concerns` doesn't really need to be part of the namespacing, at least in my opinion.